### PR TITLE
feat(api): Add the graphql configuration to the API

### PR DESCRIPTION
**Pull request by aaron-kleeen**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -51,6 +51,7 @@
     "document-register-element": "1.14.3",
     "express": "4.17.1",
     "graphql": "^15.0.0",
+    "graphql-type-json": "^0.3.2",
```
Comment (sebaskleeen) [inline package.json:54]: should we try to use exact versions? rather than `^` or `~` to avoid the issues we've had in the past?

Comment (aaron-kleeen) [reply package.json:54]: For that reason we have `package.lock`

Comment (aaron-kleeen) [reply package.json:54]: image

Comment (sebaskleeen) [reply package.json:54]: oki doks 👍

**Pull request merged**